helm/alertmanager/README.md

```diff
@@ -4,6 +4,8 @@ Installs a [Prometheus](https://prometheus.io) Alertmanager instance using the C
 
 ## TL;DR;
 
+Before installing alertmanager you should install [prometheus operator](https://github.com/coreos/prometheus-operator/blob/master/helm/prometheus-operator/README.md).
+
```

Comment:
There is a `Prerequisites` section in the README. Just add the `prometheus-operator` chart link as a second bullet.